modules/microshift-config-parameters-table.adoc

```diff
@@ -86,6 +131,78 @@ The following table explains {microshift-short} configuration YAML parameters an
 |`Managed` or `Removed`
 |Router status. Default is `Managed`.
 
+|`ingress.tuningOptions`
+|Objects
+|Specifies options for tuning the performance of ingress controller pods.
+
+|`ingress.tuningOptions.clientFinTimeout`
+|`string` with format `duration`
+|Defines how long a connection is held open while waiting for a client response to the server/backend before closing the connection. The default timeout is `1s`, which is 1 second.
+
+|`ingress.tuningOptions.clientTimeout`
+|`string` with format `duration`
+|Defines how long a connection is held open while waiting for a client response. The default timeout is `30s`, which is 30 seconds.
+
+|`ingress.tuningOptions.headerBufferBytes`
+|`integer` with the `format` of `int32`; `16384` is the minimum value when HTTP/2 is enabled
+|Describes how much memory in bytes must be reserved for `IngressController` connection sessions. Default value is `32768` in bytes.
+[NOTE]
+====
+Setting this field is generally not recommended because `headerBufferBytes` values that are too small can break the `IngressController` and `headerBufferBytes` values that are too large can cause the `IngressController` to use significantly more memory than necessary.
+====
+
+|`ingress.tuningOptions.headerBufferMaxRewriteBytes`
+|`integer` with the `format` of `int32`; `4096` is the minimum value
+|Describes how much memory in bytes must be reserved from `headerBufferBytes` for HTTP header rewriting and appending for `IngressController` connection sessions. Default value is `8192` bytes. Incoming HTTP requests are limited to the `headerBufferBytes` bytes minus the `headerBufferMaxRewriteBytes` bytes, meaning that the value of `headerBufferBytes` must be greater than the value of `headerBufferMaxRewriteBytes`.
+[NOTE]
+====
+Setting this field is generally not recommended because `headerBufferBytes` values that are too small can break the `IngressController` and `headerBufferBytes` values that are too large can cause the `IngressController` to use significantly more memory than necessary.
+====
+
+|`ingress.tuningOptions.healthCheckInterval: ""`
+|`string` with pattern: `^(0|([0-9]+(\\.[0-9]+)?(ns|us|µs|μs|ms|s|m|h))+)$`
```

Comment:
there is  something to be broken here,
in the preview the table after ingress.tuningOptions.healthCheckInterval , looks like the description of that config parsed as more configs.
![image](https://github.com/user-attachments/assets/64edbcbf-e919-432c-9857-de261433bdcd)